base/pop/lib/lib/getpoplog.p

```diff
@@ -3,6 +3,6 @@
 ;;;     - section independent
 ;;;     - safe against multiple reloads
 
-extend_searchlist( '$usepop/pop/getpoploglibs/auto', popautolist ) -> popautolist;
-extend_searchlist( '$usepop/pop/getpoploglibs/lib', popliblist ) -> popliblist;
```

Comment:
Was this a bug? Why the change from popliblist to popuseslist?

Reply:
It was a bug.